libraries/cli/lib/Configuration.ts

```diff
@@ -60,11 +60,11 @@ export class Configuration {
           config: true,
           type: "string",
         })
-        // ui
-        .option("user-interface", {
+        .option("preset", {
```

Comment:
Should we allow multiple presets? It would create some challenges w.r.t. how to resolve conflicting config options. But if we by default have a search algorithm preset enabled, all other presets become useless.

Reply:
Currently, it only allows one preset to be set.
You can add multiple preset modules, but you can only select one preset to be used.
I don't see the added benefit of having multiple presets. Can you provide an example of a use-case?

Reply:
Since the preset for the search algorithm will be on most of the time, it will make any other preset unusable. Other use cases for presets that I can think of are predefined settings for plugins, an easy way to define the settings for a research experiment (although, this would replace the search algorithm preset).

If we support multiple presets the merging would be difficult and unpredictable. We could leave it like this and just see if a use case arises that needs multiple presets.